src/Parser/CameraCalibrationParser.js

```diff
@@ -8,21 +8,17 @@ import OrientedImageCamera from 'Renderer/OrientedImageCamera';
  * @module CameraCalibrationParser
  */
 
-THREE.Matrix4.prototype.setMatrix3 = function setMatrix3(m) {
-    this.elements[0] = m.elements[0];
-    this.elements[1] = m.elements[1];
-    this.elements[2] = m.elements[2];
-    this.elements[4] = m.elements[3];
-    this.elements[5] = m.elements[4];
-    this.elements[6] = m.elements[5];
-    this.elements[8] = m.elements[6];
-    this.elements[9] = m.elements[7];
-    this.elements[10] = m.elements[8];
-    return this;
-};
-
+const xAxis = new THREE.Vector3();
+const yAxis = new THREE.Vector3();
+const zAxis = new THREE.Vector3();
 const textureLoader = new THREE.TextureLoader();
 const matrix3 = new THREE.Matrix3();
+
+THREE.Matrix4.prototype.makeBasisFromMatrix3 = function makeBasisFromMatrix3(m) {
```

Comment:
it also works with a Matrix4, so it should rather be "makeBasisFromMatrix"

Reply:
done